backend/routers/field_officer.py

```
@@ -256,138 +287,148 @@ def officer_check_out(request: OfficerCheckOutRequest, db: Session = Depends(get
+        )
 
 
-@router.get("/field-officer/issue/{issue_id}/visit-history", response_model=VisitHistoryResponse)
+@router.get(
+    "/field-officer/issue/{issue_id}/visit-history", response_model=VisitHistoryResponse
+)
 def get_issue_visit_history(
-    issue_id: int,
-    public_only: bool = True,
-    db: Session = Depends(get_db)
+    issue_id: int, public_only: bool = True, db: Session = Depends(get_db)
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)